access-modifier-checker/src/main/java/org/kohsuke/accmod/impl/Checker.java

```diff
@@ -163,7 +163,7 @@ public AnnotationVisitor visitAnnotation(String desc, boolean visible) {
 
                     @Override
                     public MethodVisitor visitMethod(int access, final String methodName, final String methodDesc, String signature, String[] exceptions) {
-                        return new EmptyVisitor() {
+                        return new MethodVisitor(Opcodes.ASM5) {
```

Comment:
AFAICT this completely disabled `visitMethodInsn` and thus broke the most common use case of this library. Unfortunately it has no test suite…